src/cijoe/cli/cli.py

```diff
@@ -428,6 +428,12 @@ def parse_args():
         const=1,
         help="Increase log-level. Provide '-l' for info and '-ll' for debug.",
     )
+    workflow_group.add_argument(
+        "--monitor",
+        "-m",
+        action="store_true",
+        help="Monitor workflow-output in the current-workdir-directory (cwd).",
```

Comment:
Perhaps rephrase this to something along the lines of: "Dump command output to stdout"?